### DISCUSSION TITLE
[Projects Beta] Shorter repository name

### DISCUSSION THREAD

**Discussion by msvab** [community/community#6073] [Projects and Issues]
In our projects we only use repositories from our own organization. In the table view, the full repository name with the organization name take up a fair amount of space which doesn't really give us any value. It would be great if there was an option to show only the repository name.

**Comment by AoDev** [comment]
I'd like to see the same. 

Reasons:

* On small screens the name is truncated and you might only see the org name.
* The repositories name plays the role of "label" in our case, being able to sort and have a better visual cue helps with decisions. (easier to read repo name than org/repo-name)
